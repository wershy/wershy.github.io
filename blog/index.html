<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link href="https://fonts.googleapis.com/css?family=Playfair+Display+SC:400,400i,700" rel="stylesheet">
    <link rel="stylesheet" href="css/style.css">
    <title>Главная страница</title>
</head>
<body>
    <header>
        <div class="contenier">
            <img class="logo" src="img/logo.png" alt="logo">
            <nav>
                <ul class="menu">
                    <li>
                        <a href="">lifestyle</a>
                    </li>
                    <li>
                        <a href="">photodiary</a>
                    </li>
                    <li>
                        <a href="">music</a>
                    </li>
                    <li>
                        <a href="">travel</a>
                    </li>

                </ul>
            </nav>
            <img src="img/image.png" alt="banner" class="banner">
            
            <h3>photodiary</h3>
            <h2>The perfect weekend getaway</h2>
                
            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem.</p>





        </div>
    </header>

    <div class="work">
            <div class="contenier">
                <div class="work__content">
                    <div class="work__content_foto">
                        <img src="img/1.png" alt="photodiary">
                        <h3>lifestyle</h3>
                        <h2>More than just a music festival </h2>
                        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea </p>

                    </div>
                    <div class="work__content_foto">
                        <img src="img/2.png" alt="photodiary">
                        <h3>lifestyle</h3>
                        <h2>Life tastes better with coffee</h2>
                        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea </p>


                    </div>
                    <div class="work__content_foto">
                        <img src="img/3.png" alt="photodiary">
                        <h3>photodiary</h3>
                        <h2>American dream</h2>
                        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea </p>


                    </div>
                    <div class="work__content_foto">
                        <img src="img/4.png" alt="photodiary">
                        <h3>photodiary</h3>
                        <h2>A day exploring the Alps</h2>
                        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea </p>


                    </div>








                </div>

            </div>

    </div>
    <div class="newsletter">
            <div class="contenier">
                    <div class="newsletter_email">
                        <h2>Sign up for our newsletter!</h2>
                        <div class="email">
                            <label for="email">Enter a email</label>
                            <input type="email" name="email" required>
                          </div>
    
                    </div>
    
    
            </div>
            
     </div>
     <div class="work">
            <div class="contenier">
                    <div class="work__content">
                            <div class="work__content_foto">
                                    <img src="img/5.png" alt="photodiary">
                                    <h3>lifestyle</h3>
                                    <h2>More than just a music festival </h2>
                                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea </p>
            
                                </div>
                                <div class="work__content_foto">
                                        <img src="img/6.png" alt="photodiary">
                                        <h3>lifestyle</h3>
                                        <h2>More than just a music festival </h2>
                                        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea </p>
                
                                    </div>

                    </div>

            </div>
         

     </div>
     <footer>
            <div class="contenier">
                    <ul class="footer_menu">
                        <li>
                            <a href="">Terms and conditions</a>
                         </li>
                         <li>
                            <a href="">Privacy</a>
                        </li>
                        <li>
                            <a href="">Follow</a>
                         </li>
                         <li>
                             <a href=""><img src="img/facebook.png" alt=""></a>
                        </li>
                        <li>
                            <a href=""><img src="img/instagram.png" alt=""></a>
                        </li>
                        <li>
                            <a href=""><img src="img/twitter.png" alt=""></a>
                        </li>
                   </ul>
    
                    

            </div>

     </footer>
     

    
</body>
</html>