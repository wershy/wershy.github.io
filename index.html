<!DOCTYPE html>
<html lang="zxx">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="x-ua-compatible" content="ie=edge">
    <title>Porfolio</title>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.0/normalize.css">
    <link rel="shortcut icon" href="img/favicon.ico">
    <link href="https://fonts.googleapis.com/css?family=Merriweather|PT+Sans+Caption:400,700" rel="stylesheet">
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
    <link rel="stylesheet" href="css/styles.css">
   


</head>

<body>
    <!-- HEADER-->
    <div class="header">
        <div class="header__title">Павел Вершинин</div>
        <div class="header_slogan">Верстка сайтов на HTML и CSS</div>
        <div class="header__arrow">
            <i class="fa fa-chevron-down" aria-hidden="true"></i>
        </div>
    </div>

    <!--   PORTFOLIO -->
    <div class="portfolio">
            <h2 class="title">Портфолио</h2>
            <div class="container">
                
                <div class="onecol">
                    <img src="img/noemi.jpg" alt="Проект Органика">
                    <h3><a href="https://pavelversh.github.io" target="_blank">Макет Noemi</a></h3>
                    <p>Верстка одностраничного сайта по PSD<br> макету</p>
                </div>
                <br><br>
                <div class="onecol">
                    <img src="img/mycompany.jpg" alt="Макет Аполло">
                    <h3><a href="https://pavelcom.github.io" target="_blank">Учебный макет My Company</a></h3>
                    <p>Верстка лендинга хостинг компании.<br />Мобильная адаптивность.</p>
                </div>
                <br><br>
                <div class="onecol">
                    <img src="img/portfolio.png" alt="Личный сайт">
                    <h3><a href="#">Личный сайт</a></h3>
                    <p>Верстка личного сайта портфолио на<br /> котором вы сейчас находитесь</p>
                </div>
            </div>
            <br><br><br><br><br><br>
        </div>

    <div class="footer">
        <div class="container">
            <div class="onecol">
                <h3>Павел Вершинин</h3>
                <p>HTML верстка и разработка сайтов</p>
            </div>
            <div class="onecol">
                <p>Мои профили в социальных сетях:</p>
                <p class="footer-icons">
                    <a href="#" target="_blank"><i class="fa fa-vk" aria-hidden="true"></i></a>
                    <a href="#" target="_blank"><i class="fa fa-facebook-official" aria-hidden="true"></i></a>
                    <a href="#" target="_blank"><i class="fa fa-instagram" aria-hidden="true"></i></a>
                    <a href="#" target="_blank"><i class="fa fa-twitter" aria-hidden="true"></i></a>
                    <a href="#" target="_blank"><i class="fa fa-youtube" aria-hidden="true"></i></a>
                </p>
            </div>
            <div class="onecol_button">
               <div> <a href="https://vk.com" class="button" target="_blank">Связаться ВКОНТАКТЕ</a></div>
                <p>Напишите мне, чтобы заказать или узнать<br> стоимость верстки вашего проекта</p>
            </div>
        </div>
    </div>
    
    
</body>

</html>
