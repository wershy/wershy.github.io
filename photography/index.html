<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet">

    <link rel="stylesheet" href="css/styles.css">
    <link rel="stylesheet" href="css/blueimp-gallery.min.css">
    <title>Document</title>
</head>
<body>
    <header id="home">
            <div class="contanier">
                    <nav>
                            <ul class="menu">
                                <li>
                                    <a class="nav_link" href="#home">Home</a>
                                </li>
                                <li>
                                    <a class="nav_link" href="#about">About Me</a>
                                </li>
                                <li>
                                    <a class="nav_link" href="#portfolio">Portfolio</a>
                                </li>
                                <li>
                                    <a class="nav_link" href="#contact">Contact</a>
                                </li>
                                
                            </ul>
                        </nav>
                <img src="img/camera.png" alt="" class="logo">   
                <h1><strong>#Jwood</strong>PHOTOGRAPHY</h1>  
                
            </div>
            

    </header>
    <div class="about" id="about">
            <div class="contanier">
                    <ul class="menu">
                            <li>
                                <a class="nav_link_2" href="#home">Home</a>
                            </li>
                            <li>
                                <a class="nav_link_2" href="#about">About Me</a>
                            </li>
                            <li>
                                <a class="nav_link_2" href="#portfolio">Portfolio</a>
                            </li>
                            <li>
                                <a  class="nav_link_2" href="#contact">Contact</a>
                            </li>
                            
                        </ul>
            </div>
            <div class="contanier_2">
                <div class="about_me">
                    
                    <div class="about_me_text">
                        <h2>About Me</h2>
                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.<br><br> Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur  Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. <br><br>


                                Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.</p>
                    </div>
                    <img src="img/JasonWood.png" alt="Jason Wood" class="about_me__photo">

                </div>
                
            </div>    
    </div>
    <div class="portfolio" id="portfolio">
            <div class="contanier">
                    <ul class="menu">
                            <li>
                                <a class="nav_link_2" href="#home">Home</a>
                            </li>
                            <li>
                                <a class="nav_link_2" href="#about">About Me</a>
                            </li>
                            <li>
                                <a class="nav_link_2" href="#portfolio">Portfolio</a>
                            </li>
                            <li>
                                <a  class="nav_link_2" href="#contact">Contact</a>
                            </li>
                            
                        </ul>

            </div>
            <div class="contanier_2">
                <h2 class="Port_text">Portfolio</h2>
                <div class="footer-gallery" id="links">
                    
                    <div class="footer-gallery_thumb">
                            <a href="img/big01.png"> <img src="img/01.png" alt="Photo"></a>
                    </div>
                    <div class="footer-gallery_thumb">
                            <a href="img/big02.png"> <img src="img/02.png" alt="Photo"></a>
                    </div>
                    <div class="footer-gallery_thumb">
                            <a href="img/big03.png">  <img src="img/03.png" alt="Photo"></a>
                    </div>
                    <div class="footer-gallery_thumb">
                            <a href="img/big04.png">  <img src="img/04.png" alt="Photo"></a>
                    </div>
                    <div class="footer-gallery_thumb">
                            <a href="img/big05.png"> <img src="img/05.png" alt="Photo"></a>
                    </div> 
                    <div class="footer-gallery_thumb">
                            <a href="img/big06.png"> <img src="img/06.png" alt="Photo"></a>
                    </div>  
                    <div class="footer-gallery_thumb">
                            <a href="img/big07.png"> <img src="img/07.png" alt="Photo"></a>
                    </div> 
                    <div class="footer-gallery_thumb">
                            <a href="img/big08.png"> <img src="img/08.png" alt="Photo"></a>
                    </div>
                    <div class="footer-gallery_thumb">
                            <a href="img/big09.png"> <img src="img/09.png" alt="Photo"></a>
                    </div>
                    <div class="footer-gallery_thumb">
                            <a href="img/big10.png"> <img src="img/10.png" alt="Photo"></a>
                    </div>
                </div>
                <div id="blueimp-gallery" class="blueimp-gallery">
                        <div class="slides"></div>
                        <h3 class="title"></h3>
                        <a class="prev">‹</a>
                        <a class="next">›</a>
                        <a class="close">×</a>
                        <a class="play-pause"></a>
                        <ol class="indicator"></ol>
                </div>
                
            </div>

    </div>
    <footer id="contact">
            <div class="contanier">
                    <ul class="menu">
                            <li>
                                <a class="nav_link_2" href="#home">Home</a>
                            </li>
                            <li>
                                <a class="nav_link_2" href="#about">About Me</a>
                            </li>
                            <li>
                                <a class="nav_link_2" href="#portfolio">Portfolio</a>
                            </li>
                            <li>
                                <a  class="nav_link_2" href="#contact">Contact</a>
                            </li>
                            
                        </ul>
                        <div class="contacts">
                            <h2>Mail:</h2>
                            <p>Jason@wood.com</p>
                            <h2>Phone</h2>
                            <p>+44 212 432 444</p>
                        
                            <h2>social media:</h2>
                            <img src="img/Facebook.png" alt="">
                            <img src="img/Instagram.png" alt="">
                            <img src="img/Twitter.png" alt="">
                        </div>

            </div>
    </footer>
    <script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>	
    <script src="js/blueimp-gallery.min.js"></script>  
    <script>
            document.getElementById('links').onclick = function (event) {	
                   event = event || window.event;
                   var target = event.target || event.srcElement,
                       link = target.src ? target.parentNode : target,
                       options = {index: link, event: event},
                       links = this.getElementsByTagName('a');
                   blueimp.Gallery(links, options);
            };
            </script>  
</body>
</html>