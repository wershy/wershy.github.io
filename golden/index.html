<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link rel="stylesheet" href="css/styles.css">
    <title>golden</title>
</head>
<body>
    <header>
        <div class="contanier">
            <div class="heading clearfix">
                <img class="logo" src="img/logo.png" alt="logo">
                <nav>
                    <ul class="menu">
                        <li>
                            <a href="#">Home</a>
                        </li>
                        <li>
                            <a href="#">Services</a>
                        </li>
                        <li>
                            <a href="#">Portfolio</a>
                        </li>
                        <li>
                            <a href="#">About</a>
                        </li>
                        <li>
                            <a href="#">Contact</a>
                        </li>
                    </ul>
                </nav>
            </div> 
            <div class="titles">
                <div class="titles__first">Welcome To Our Studio!</div>
                <h1>It’s nice to Meet you</h1>
            </div> 
            <a class="button" href="#">Tell Me More</a>  

        </div>
    </header>
    <section id="services">
            <div class="contanier">
              <div class="title">
                <h2>Services</h2>
                <p>Proin iaculis purus consequat sem cure. </p>
              </div> 
              <div class="services clearfix">
                  <div class="services__item">
                    <img src="img/icon1.png" alt="услуга">
                    <h3>E-Commerce</h3>
                    <p>Proin iaculis purus consequat sem cure 
                            digni ssim. Donec porttitora entum suscipit 
                            aenean rhoncus posuere odio in tincidunt.</p>
                  </div>
                  <div class="services__item">
                    <img src="img/icon2.png" alt="Responsive Web">
                    <h3>Responsive Web</h3>
                    <p>Proin iaculis purus consequat sem cure 
                                digni ssim. Donec porttitora entum suscipit 
                                aenean rhoncus posuere odio in tincidunt.</p>

                  </div>
                  <div class="services__item">
                    <img src="img/icon3.png" alt="Web Security">
                    <h3>Web Security</h3>
                    <p>Proin iaculis purus consequat sem cure 
                                digni ssim. Donec porttitora entum suscipit 
                                aenean rhoncus posuere odio in tincidunt.</p>

                  </div>
              </div> 
            </div>
    </section>
    <section id="porfolio">
            <div class="contanier">
                    <div class="title">
                            <h2>Our portfolios</h2>
                            <p>Proin iaculis purus consequat sem cure. </p>

                    </div>
                    <div class="works clearfix">
                        <img src="img/port1.png" alt="works">
                        <img src="img/port2.png" alt="works">
                        <img src="img/port3.png" alt="works">
                    </div>
            
            </div>
    </section>
    <section id="about">
            <div class="contanier">
                    <div class="title">
                            <h2>About Us</h2>
                            <p>Proin iaculis purus consequat sem cure. </p>
                            
                    </div>
                    <div class="services clearfix">
                            <div class="services__item">
                              <img src="img/about1.png" alt="услуга">
                              <h3>Our Humble Beginnings</h3>
                              <p>Proin iaculis purus consequat sem cure 
                                      digni ssim. Donec porttitora entum suscipit 
                                      aenean rhoncus posuere odio in tincidunt.</p>
                            </div>
                            <div class="services__item">
                              <img src="img/about2.png" alt="Responsive Web">
                              <h3>Facing Startup Battles</h3>
                              <p>Proin iaculis purus consequat sem cure 
                                          digni ssim. Donec porttitora entum suscipit 
                                          aenean rhoncus posuere odio in tincidunt.</p>
          
                            </div>
                            <div class="services__item">
                              <img src="img/about3.png" alt="Web Security">
                              <h3>Enter The Dark Days</h3>
                              <p>Proin iaculis purus consequat sem cure 
                                          digni ssim. Donec porttitora entum suscipit 
                                          aenean rhoncus posuere odio in tincidunt.</p>
          
                            </div>
                            
                    </div> 
            </div>
    </section>
    <footer id="copy">
            <div class="contanier">
                    <div class="title">
                        <p>© Copyright 2014 FreebiesXpress.com</p>
                    </div>
            
            </div>
    </footer>
    
</body>
</html>